application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -122,8 +123,8 @@ adminTools.dashboard.section.security.title=Security
         <number>1</number>
         <prettyName>Scope</prettyName>
         <relationalStorage>0</relationalStorage>
-        <separator/>
-        <separators>|,</separators>
+        <separator> </separator>
+        <separators>|, </separators>
```

Comment:
These do no seem related to this PR. If some xml changes where applied during `mvn xar:format` because of the xwiki parent change, than these should be added along with the commit that updated the parent. If the changes were applied anyway, even with the old parent, than it means that:
1. the pages where exported from another XWiki version than the minimal one (14.10) 
2. what we have on master are not pages exported from the minimal supported XWiki version
3. you did some local changes and committed them by mistake

Reply:
Yes, the used version was actually 14.10.2 (as parent version 14.10 actually brings 14.10.2), switched now to 14.10 by using parent 14.10-1. I will make a commit into master for changing the version of the parent to 14.10-1 and also with the modifications made by `mvn xar:format`. Would that be ok?